win32ss/user/ntuser/msgqueue.c

```diff
@@ -1487,6 +1487,12 @@ BOOL co_IntProcessMouseMessage(MSG* msg, BOOL* RemoveMessages, BOOL* NotForUs, L
 
     pti = PsGetCurrentThreadWin32Thread();
     pwndDesktop = UserGetDesktopWindow();
+    if (pwndDesktop == NULL)
```

Comment:
```suggestion
    if (!pwndDesktop)
```

Reply:
No, it's a pointer, not a boolean value. It should be compared to NULL.